packages/firebase_crashlytics/firebase_crashlytics/ios/crashlytics_add_upload_symbols

```diff
@@ -0,0 +1,35 @@
+require 'xcodeproj'
+
+if(ARGV.length < 2) 
+    abort("Not enough arguments provided. Please pass at least a project directory and project name.")
+end
+
+current_dir = ARGV[0]
+project_name = ARGV[1]
+
+project_path = File.join(current_dir, project_name)
+puts(project_path)
+unless (File.exist?(project_path)) 
+    abort("Project at #{project_path} does not exist. Please incorporate Crashlytics upload symbols manually.\n");
+end
+
+# Default to Flutter arguments if no other upload-symbols inputs are provided.
+if(ARGV.length == 3)
+    upload_symbols_args = ARGV[2]
+else
+    upload_symbols_args = "--flutter-project \"$PROJECT_DIR/firebase_app_id_file.txt"
```

Comment:
Is this .txt file in relation to the CLI PR https://github.com/invertase/flutterfire_cli/pull/43?

Reply:
Yes - I changed the file format to be more structured (now is json) so need to update this!